modules/security-container-content-external-scanning.adoc

```diff
@@ -0,0 +1,262 @@
+    { "label": "moderate", "data": "8", "severityIndex": 1, "reference": null },
+    { "label": "low", "data": "26", "severityIndex": 0, "reference": null }
+  ]
+}
+----
+
+This example shows the
+link:https://catalog.redhat.com/software/containers/explore[Red Hat Container Catalog]
+annotation for an image with health index data
+with an external URL for additional details:
+
+.Red Hat Container Catalog Annotation
+[source,json]
+----
+{
+  "name": "Red Hat Container Catalog",
+  "description": "Container health index",
+  "timestamp": "2016-09-08T05:04:46Z",
+  "reference": "https://access.redhat.com/errata/RHBA-2016:1566",
+  "compliant": null,
+  "scannerVersion": "1.2",
+  "summary": [
+    { "label": "Health index", "data": "B", "severityIndex": 1, "reference": null }
+  ]
+}
+----
+
+[[security-annotating-image-objects]]
+== Annotating image objects
+
+While image stream objects
+are what an end-user of {product-title} operates against,
+image objects are annotated with
+security metadata. Image objects are cluster-scoped, pointing to a single image
+that may be referenced by many image streams and tags.
+
+[[security-example-annotate-CLI]]
+=== Example annotate CLI command
+
+Replace `<image>` with an image digest, for example
+`sha256:401e359e0f45bfdcf004e258b72e253fd07fba8cc5c6f2ed4f4608fb119ecc2`:
+
+----
+$ oc annotate image <image> \
+    quality.images.openshift.io/vulnerability.redhatcatalog='{ \
+    "name": "Red Hat Container Catalog", \
+    "description": "Container health index", \
+    "timestamp": "2020-06-01T05:04:46Z", \
+    "compliant": null, \
+    "scannerVersion": "1.2", \
+    "reference": "https://access.redhat.com/errata/RHBA-2020:2347", \
+    "summary": "[ \
+      { "label": "Health index", "data": "B", "severityIndex": 1, "reference": null } ]" }'
+----
+
+[[controlling-pod-execution]]
+== Controlling pod execution
+
+To programmatically control if an image may be run, the
+`images.openshift.io/deny-execution` image policy may be used.
+
+[[security-controlling-pod-execution-example-annotation]]
+=== Example annotation
+
+[source,yaml]
+----
+annotations:
+  images.openshift.io/deny-execution: true
+----
+
+[[security-integration-reference]]
+== Integration reference
+
+In most cases, external tools such as vulnerability scanners will develop a
+script or plug-in that watches for image updates, performs scanning, and
+annotates the associated image object with the results. Typically this
+automation calls the {product-title} REST API to write the annotation. See
+the REST API Reference for general
+information on the REST API and `PATCH` call to update images.
+
+[[security-integration-reference-example-api-call]]
+=== Example REST API call
+
+The following example call using `curl` overrides the value of the
+annotation. Be sure to replace the values for `<token>`, `<openshift_server>`,
+`<image_id>`, and `<image_annotation>`.
+
+.Patch API Call
```

Comment:
A lot of these headings on examples are title case and need to be sentence case, if you can look through the PR to update the rest

Reply:
Okay. Fixed.